# Copyright (C) 2011 THE plugins'S COPYRIGHT HOLDER
# This file is distributed under the same license as the plugins package.
# Automatically generated, 2011.
#
msgid ""
msgstr ""
"Project-Id-Version: enigma2-vision-core\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-08-03 03:15+0430\n"
"PO-Revision-Date: 2019-07-04 23:44+0200\n"
"Last-Translator: Kojo\n"
"Language-Team: Samzan/Kojo\n"
"Language: fi\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Poedit 2.2\n"

#: ../src/BackupManager.py:158 ../src/BackupManager.py:292
#: ../src/ImageManager.py:151 ../src/ImageManager.py:291
msgid "%a %e %b  %-H:%M"
msgstr "%a %e %b  %-H:%M"

#: ../src/SwapManager.py:302
msgid "128 Mb"
msgstr "128 Mb"

#: ../src/SwapManager.py:302
msgid "16 Mb"
msgstr "16 Mb"

#: ../src/SwapManager.py:302
msgid "256 Mb"
msgstr "256 Mb"

#: ../src/BackupManager.py:52 ../src/ImageManager.py:62
msgid "30 Days"
msgstr "30 päivää"

#: ../src/SwapManager.py:302
msgid "32 Mb"
msgstr "32 Mb"

#: ../src/SwapManager.py:302
msgid "64 Mb"
msgstr "64 Mb"

#: ../src/SwapManager.py:302
msgid "8 Mb"
msgstr "8 Mb"

#: ../src/SwapManager.py:302
msgid "96 Mb"
msgstr "96 Mb"

#: ../src/BackupManager.py:504 ../src/RestoreWizard.py:156
#: ../src/RestoreWizard.py:267
msgid "A background update check is in progress, please try again."
msgstr "Taustalla ajettava päivitystarkistus käynnissä, yritä uudelleen."

#: ../src/SwapManager.py:120 ../src/SwapManager.py:243
msgid "Activate"
msgstr "Aktivoi"

#: ../src/SwapManager.py:118 ../src/SwapManager.py:239
msgid "Active"
msgstr "Aktiivinen"

#: ../src/SoftcamManager.py:83 ../src/SoftcamManager.py:149
msgid "Active:"
msgstr "Aktiivinen:"

#: ../src/IPKInstaller.py:124
msgid "Are you ready to install ?"
msgstr "Oletko valmiina asentamaan?"

#: ../src/Multibootmgr.py:85
#, python-format
msgid "Are you sure you want to delete image slot %s ?"
msgstr "Oletko varma, että haluat poistaa imagepaikan %s ?"

#: ../src/BackupManager.py:300
msgid ""
"Are you sure you want to delete this backup:\n"
" "
msgstr ""
"Haluatko poistaa tämän varmuuskopion:\n"
" "

#: ../src/ImageManager.py:299
msgid ""
"Are you sure you want to delete this image backup:\n"
" "
msgstr ""
"Oletko varma että haluat poistaa tämän varmuuskopion:\n"
" "

#: ../src/ImageManager.py:1459
msgid ""
"Are you sure you want to download this image:\n"
" "
msgstr ""
"Haluatko varmasti ladata tämän ohjelmiston:\n"
" "

#: ../src/ImageManager.py:375
msgid ""
"Are you sure you want to flash this image:\n"
" "
msgstr ""
"Oletko varma että haluat asentaa tämän imagen:\n"
" "

#: ../src/BackupManager.py:363
msgid ""
"Are you sure you want to restore this backup:\n"
" "
msgstr ""
"Oletko varma, että haluat palauttaa tämän varmuuskopion:\n"
" "

#: ../src/SoftcamManager.py:108 ../src/SwapManager.py:122
msgid "Autostart"
msgstr "Autom.käynnistys"

#: ../src/ImageManager.py:767
msgid "Backing up eMMC partitions for USB flash ..."
msgstr "Varmuuskopioidaan eMMC-osioita USB:lle..."

#: ../src/ImageManager.py:771
msgid "Backing up eMMC partitions for USB flash..."
msgstr "Varmuuskopioidaan eMMC-osioita USB:lle..."

#: ../src/BackupManager.py:1118 ../src/BackupManager.py:1122
#: ../src/BackupManager.py:1134 ../src/BackupManager.py:1138
msgid "Backing up files..."
msgstr "Varmuuskopioidaan tiedostoja..."

#: ../src/ImageManager.py:751 ../src/ImageManager.py:755
msgid "Backing up kernel..."
msgstr "Varmuuskopioidaan kernel..."

#: ../src/ImageManager.py:759 ../src/ImageManager.py:763
msgid "Backing up root file system..."
msgstr "Varmuuskopioidaan järjestelmätiedostot..."

#: ../src/ImageManager.py:799
msgid "Backup complete."
msgstr "Varmuuskopio valmis."

#: ../src/BackupManager.py:1142
msgid "Backup complete..."
msgstr "Varmuuskopio valmis..."

#: ../src/ImageManager.py:330
msgid "Backup confirmation"
msgstr "Varmuuskopion vahvistus"

#: ../src/BackupManager.py:937
msgid "Backup created"
msgstr "Varmuuskopio luotu"

#: ../src/BackupManager.py:1305
msgid "Backup failed - e. g. wrong backup destination or no space left on backup device."
msgstr "Varmuuskopio epäonnistui - Saattaa johtua väärästä kohdesijainnista, tai levytilaa on liian vähän."

#: ../src/BackupManager.py:347
msgid ""
"Backup in progress,\n"
"Please wait for it to finish, before trying again."
msgstr ""
"Varmuuskopiointi käynnissä, \n"
"Odota että se loppuu, ennen kuin yrität uudestaan."

#: ../src/BackupManager.py:137 ../src/BackupManager.py:184
#: ../src/BackupManager.py:316 ../src/BackupManager.py:331
#: ../src/BackupManager.py:374 ../src/BackupManager.py:392
#: ../src/BackupManager.py:898 ../src/BackupManager.py:1100 ../src/ui.py:48
#: ../src/ImageManager.py:350
msgid "Backup manager"
msgstr "Varmuuskopioiden hallinnointi"

#: ../src/MountManager.py:282
msgid "Can't un-mount the partition; make sure it is not being used for SWAP or record/timeshift paths."
msgstr "Liitosta ei voi purkaa; tarkista että SWAP-tiedosto, tallennus, tai ajansiirto ei käytä sitä."

#: ../src/BackupManager.py:719 ../src/BackupManager.py:808
#: ../src/BackupManager.py:908 ../src/Multibootmgr.py:36
#: ../src/MountManager.py:337
msgid "Cancel"
msgstr "Keskeytä"

#: ../src/IPKInstaller.py:88 ../src/IPKInstaller.py:92
msgid "Change location"
msgstr "Vaihda sijainti"

#: ../src/ImageManager.py:743
msgid "Checking free RAM.."
msgstr "Tarkistetaan vapaana oleva RAM..."

#: ../src/SoftcamManager.py:557
msgid "Checking softcams..."
msgstr "Tarkistetaan softcameja..."

#: ../src/BackupManager.py:910
msgid "Choose files"
msgstr "Valitse tiedostot"

#: ../src/BackupManager.py:911
msgid "Choose local IPK's folder"
msgstr "Paikallinen IPK-kansio"

#: ../src/MountManager.py:334
msgid "Choose where to mount your devices to:"
msgstr "Valitse minne laitteet liitetään:"

#: ../src/ui.py:57 ../src/ImageManager.py:1320 ../src/SoftcamManager.py:105
#: ../src/IPKInstaller.py:54 ../src/IPKInstaller.py:176
#: ../src/SwapManager.py:119
msgid "Close"
msgstr "Sulje"

#: ../src/BackupManager.py:414 ../src/BackupManager.py:418
msgid "Comparing against backup..."
msgstr "Vertaillaan varmuuskopioon..."

#: ../src/ImageManager.py:1427
msgid "Compress"
msgstr "Pakkaa"

#: ../src/SwapManager.py:121 ../src/SwapManager.py:183
msgid "Create"
msgstr "Luo"

#: ../src/ui.py:53
msgid "Create and Manage your SWAP files."
msgstr "Luo ja hallinnoi SWAP-tiedostoja."

#: ../src/ui.py:49
msgid "Create and flash complete images of your system."
msgstr "Luo ja asenna image koko järjestelmästäsi."

#: ../src/ImageManager.py:947 ../src/ImageManager.py:950
#: ../src/ImageManager.py:953 ../src/ImageManager.py:955
#: ../src/ImageManager.py:957
msgid "Create:"
msgstr "Luo:"

#: ../src/ImageManager.py:747
msgid "Creating SWAP.."
msgstr "Luodaan sivutustiedostoa..."

#: ../src/BackupManager.py:406 ../src/BackupManager.py:410
#: ../src/BackupManager.py:1110 ../src/BackupManager.py:1114
msgid "Creating list of installed plugins..."
msgstr "Luodaan luettelo asennetuista lisäosista..."

#: ../src/ImageManager.py:791 ../src/ImageManager.py:795
msgid "Creating zip..."
msgstr "Luodaan zip..."

#: ../src/SwapManager.py:145 ../src/SwapManager.py:239
#: ../src/SwapManager.py:244
msgid "Current status:"
msgstr "Nykyinen tila:"

#: ../src/BackupManager.py:52 ../src/ImageManager.py:62
msgid "Daily"
msgstr "Päivittäin"

#: ../src/SwapManager.py:238
msgid "Deactivate"
msgstr "Poista käytöstä"

#: ../src/BackupManager.py:143 ../src/ImageManager.py:129
#: ../src/SwapManager.py:193
msgid "Delete"
msgstr "Poista"

#: ../src/BackupManager.py:753
msgid "Deselect"
msgstr "Poista valinta"

#: ../src/BackupManager.py:235 ../src/BackupManager.py:255
#: ../src/ImageManager.py:244 ../src/ImageManager.py:253
#: ../src/MountManager.py:144 ../src/MountManager.py:260
#: ../src/MountManager.py:275
msgid "Device: "
msgstr "Laite: "

#: ../src/ImageManager.py:221
msgid "Device: None available"
msgstr "Laite: ei saatavilla"

#: ../src/BackupManager.py:216
msgid "Device: none available"
msgstr "Laite: Ei yhtään saatavilla"

#: ../src/SoftcamManager.py:145
msgid "Disable startup"
msgstr "Käynnistys pois päältä"

#: ../src/ImageManager.py:261
msgid "Do you want to change download url"
msgstr "Haluatko vaihtaa latausosoitteen?"

#: ../src/ImageManager.py:328
msgid ""
"Do you want to create a full image backup?\n"
"This can take about 6 minutes to complete."
msgstr ""
"Haluatko ottaa varmuuskopion imagesta?\n"
"Tämä saattaa kestää noin kuusi minuuttia."

#: ../src/ImageManager.py:371
#, python-format
msgid ""
"Do you want to flash image\n"
"%s"
msgstr ""
"Haluatko asentaa imagen\n"
"%s"

#: ../src/IPKInstaller.py:137
msgid "Do you want to restart GUI now ?"
msgstr "Haluatko käynnistää käyttöliittymän nyt uudelleen?"

#: ../src/BackupManager.py:651
msgid "Do you want to restore your Enigma2 plugins ?"
msgstr "Haluatko palauttaa käyttämäsi Enigma2-lisäosat?"

#: ../src/BackupManager.py:446
msgid "Do you want to restore your enigma2 settings ?"
msgstr "Haluatko palauttaa enigma2 -asetuksesi?"

#: ../src/ImageManager.py:1429
msgid "DownLoad"
msgstr "Lataa"

#: ../src/ImageManager.py:1321
msgid "Download"
msgstr "Lataa"

#: ../src/ImageManager.py:1461
msgid "Download confirmation"
msgstr "Latauksen varmistus"

#: ../src/ImageManager.py:1474
msgid "Downloading"
msgstr "Lataa"

#: ../src/ImageManager.py:131 ../src/ImageManager.py:1315
msgid "Downloads"
msgstr "Lataukset"

#: ../src/ImageManager.py:432
msgid "ET8500 Image Restore"
msgstr "ET8500 Imagen palautus"

#: ../src/ImageManager.py:430
msgid ""
"ET8500 Multiboot: Yes to restore OS1 No to restore OS2:\n"
" "
msgstr ""
"ET8500 Multiboot: Kyllä, palauttaaksesi OS1,  Ei, palauttaaksesi OS2:\n"
" "

#: ../src/Multibootmgr.py:72
msgid "Empty slot"
msgstr "Tyhjä paikka"

#: ../src/SwapManager.py:246
msgid "Enable SWAP at startup"
msgstr "SWAP päälle käynnistyksessä"

#: ../src/SoftcamManager.py:147
msgid "Enable startup"
msgstr "Käynnistys päälle"

#: ../src/BackupManager.py:1064 ../src/ImageManager.py:621
msgid "Enough retries, delaying till next schedule."
msgstr "Tarpeeksi yrityksiä, koitetaan uudestaan seuraavalla kerralla."

#: ../src/Multibootmgr.py:39
msgid "Erase"
msgstr "Pyyhi"

#: ../src/RestoreWizard.py:90
msgid "Exit the restore wizard"
msgstr "Sulje palautusohjelma"

#: ../src/ImageManager.py:1427
msgid "Expand"
msgstr "Laajenna"

#: ../src/IPKInstaller.py:83 ../src/IPKInstaller.py:102
msgid "Extra IPK's"
msgstr "Lisä IPK:t"

#: ../src/ImageManager.py:132
msgid "Flash"
msgstr "Flash"

#: ../src/ImageManager.py:377
msgid "Flash confirmation"
msgstr "Päivityksen vahvistus"

#: ../src/ImageManager.py:428
msgid "Flash image unzip successful."
msgstr "Flash-imagen purku onnistui."

#: ../src/SoftcamManager.py:222 ../src/SoftcamManager.py:266
msgid "Found non-standard softcam, trying to start, this may fail."
msgstr "Löytyi epästandardi softcam. Yritetään käynnistystä, mutta tämä saattaa epäonnistua."

#: ../src/ImageManager.py:244
msgid "Free space:"
msgstr "Vapaa tila:"

#: ../src/MountManager.py:131
msgid "GB"
msgstr "GB"

#: ../src/H9SDmanager.py:31
msgid "H9 SDcard manager"
msgstr "H9 SD-kortin hallinnointi"

#: ../src/ui.py:55
msgid "H9SDcard Manager"
msgstr "H9SD-kortin hallinnointi"

#: ../src/MountManager.py:84
msgid "HARD DISK: "
msgstr "Kiintolevy: "

#: ../src/IPKInstaller.py:39 ../src/IPKInstaller.py:169
msgid "IPK installer"
msgstr "IPK:n asennus"

#: ../src/ImageManager.py:1364
#, python-format
msgid "Image Version %s"
msgstr "Imagen versio %s"

#: ../src/ui.py:49 ../src/ImageManager.py:171 ../src/ImageManager.py:319
#: ../src/ImageManager.py:340 ../src/ImageManager.py:737
msgid "Image manager"
msgstr "Imagen hallinnointi"

#: ../src/SwapManager.py:117 ../src/SwapManager.py:244
msgid "Inactive"
msgstr "Ei käytössä"

#: ../src/SwapManager.py:263 ../src/SwapManager.py:329
msgid "Info"
msgstr "Tietoja"

#: ../src/H9SDmanager.py:34
msgid "Init SDcard"
msgstr "Alusta SDcard"

#: ../src/H9SDmanager.py:45
msgid "Init Zgemma H9 SDCARD - please reboot after use."
msgstr "Alusta Zgemma H9 SD-kortti - Vaatii uudelleenkäynnistyksen."

#: ../src/H9SDmanager.py:69
msgid "Init Zgemma H9 USB/SDA1 - please reboot after use."
msgstr "Alusta Zgemma H9 USB/SDA1 - vaatii uudelleenkäynnistyksen."

#: ../src/IPKInstaller.py:55 ../src/IPKInstaller.py:177
msgid "Install"
msgstr "Asenna"

#: ../src/ui.py:50
msgid "Install IPK's from your tmp folder."
msgstr "Asenna IPK-paketteja laitteen tmp-kansiosta."

#: ../src/IPKInstaller.py:126
msgid "Install confirmation"
msgstr "Asennuksen vahvistus"

#: ../src/plugin.py:157
msgid "Install extensions."
msgstr "Asenna laajennukset."

#: ../src/ui.py:50
msgid "Install local extension"
msgstr "Asenna paikallinen lisäosa"

#: ../src/IPKInstaller.py:134
msgid "Installing..."
msgstr "Asennetaan..."

#: ../src/IPKInstaller.py:179
msgid "Invert"
msgstr "Käänteinen"

#: ../src/IPKInstaller.py:86
msgid "It seems you have not setup an extra location. Please set it up in the Backup manager setup menu."
msgstr "Et ole asettanut ylimääräistä sijaintia. Aseta se Varmuuskopioiden hallintatyökalulla."

#: ../src/BackupManager.py:934 ../src/SoftcamManager.py:478
msgid "Logs"
msgstr "Lokit"

#: ../src/ImageManager.py:244 ../src/MountManager.py:133
msgid "MB"
msgstr "MB"

#: ../src/ui.py:48
msgid "Manage the backups of your settings."
msgstr "Muokkaa asetusten varmuuskopioita."

#: ../src/ui.py:51
msgid "Manage your devices mount points."
msgstr "Muokkaa laitteen liitoksia."

#: ../src/MountManager.py:200
msgid "Mount"
msgstr "Liitä"

#: ../src/MountManager.py:265
msgid "Mount failed."
msgstr "Liitos epäonnistui."

#: ../src/ui.py:51 ../src/MountManager.py:195
msgid "Mount manager"
msgstr "Liitosten hallinnointi"

#: ../src/MountManager.py:144 ../src/MountManager.py:259
#: ../src/MountManager.py:274
msgid "Mount: "
msgstr "Liitos: "

#: ../src/ui.py:55
msgid "Move Nand root to SD card"
msgstr "Siirrä Nand root SD-kortille"

#: ../src/ImageManager.py:783 ../src/ImageManager.py:787
msgid "Moving to backup Location..."
msgstr "Siirretään varmuuskopiohakemistoon..."

#: ../src/ImageManager.py:504
msgid "Multiboot ERROR! - no STARTUP in boot partition."
msgstr "Multiboot-virhe! - Käynnistys-osiota ei ole olemassa."

#: ../src/Multibootmgr.py:35
msgid "Multiboot image manager"
msgstr "Multiboot-imagen hallinnointi"

#: ../src/ImageManager.py:436 ../src/ImageManager.py:1219
msgid "Multiboot only able to restore this backup to mmc slot1"
msgstr "Multiboot voi palauttaa tämän varmuuskopion vain mmc:lle paikkaan 1"

#: ../src/RestoreWizard.py:101 ../src/RestoreWizard.py:104
msgid "NO, do not restore plugins"
msgstr "EI, älä palauta lisäosia"

#: ../src/RestoreWizard.py:94
msgid "NO, do not restore settings"
msgstr "EI, älä palauta asetuksia"

#: ../src/BackupManager.py:189 ../src/ImageManager.py:176
msgid "New backup"
msgstr "Uusi varmuuskopio"

#: ../src/BackupManager.py:158 ../src/BackupManager.py:160
#: ../src/BackupManager.py:292 ../src/BackupManager.py:294
#: ../src/ImageManager.py:151 ../src/ImageManager.py:153
#: ../src/ImageManager.py:291 ../src/ImageManager.py:293
msgid "Next backup: "
msgstr "Seuraava varmuuskopio: "

#: ../src/SoftcamManager.py:218
#, fuzzy
msgid ""
"No config files found, please setup Ncam first\n"
"in /etc/tuxbox/config"
msgstr ""
"Asetustiedostoja ei löytynyt, määritä Oscam\n"
"/etc/tuxbox/config-tiedostossa"

#: ../src/SoftcamManager.py:264
#, fuzzy
msgid ""
"No config files found, please setup Ncam first\n"
"in /etc/tuxbox/config."
msgstr ""
"Asetus-tiedostoa ei löytynyt, asenna Oscam-asetukset\n"
"hakemistoon /etc/tuxbox/config."

#: ../src/SoftcamManager.py:213
msgid ""
"No config files found, please setup Oscam first\n"
"in /etc/tuxbox/config"
msgstr ""
"Asetustiedostoja ei löytynyt, määritä Oscam\n"
"/etc/tuxbox/config-tiedostossa"

#: ../src/SoftcamManager.py:260
msgid ""
"No config files found, please setup Oscam first\n"
"in /etc/tuxbox/config."
msgstr ""
"Asetus-tiedostoa ei löytynyt, asenna Oscam-asetukset\n"
"hakemistoon /etc/tuxbox/config."

#: ../src/ImageManager.py:1330
msgid "No images found for selected download server...if password check validity"
msgstr "Valitulta palvelimelta ei löytynyt imagea, jos vaatii salasanan, varmista että se on oikein"

#: ../src/MountManager.py:109 ../src/MountManager.py:111
#: ../src/MountManager.py:122
msgid "None"
msgstr "Ei mitään"

#: ../src/Multibootmgr.py:38
msgid "Note: slot list does not show current image or empty slots."
msgstr "HUOM: Paikkalistaus ei näytä nykyistä imagea, tai tyhjiä paikkoja."

#: ../src/BackupManager.py:668
msgid "Now skipping restore process"
msgstr "Ohitetaan palautusprosessia"

#: ../src/BackupManager.py:909 ../src/RestoreWizard.py:108
msgid "OK"
msgstr "OK"

#: ../src/RestoreWizard.py:90
msgid "OK, to perform a restore"
msgstr "OK, suorita palautus"

#: ../src/plugin.py:175
msgid "Opkg"
msgstr ""

#: ../src/BackupManager.py:864
msgid "Please enter a folder that contains some packages."
msgstr "Valitse hakemisto joka sisältää asennuspaketteja."

#: ../src/SwapManager.py:295
msgid "Please select device to use as SWAP file location."
msgstr "Valitse laite, jota käytetään SWAP-tiedoston tallentamiseen."

#: ../src/RestoreWizard.py:140
msgid "Please wait while plugins restore completes..."
msgstr "Odota, lisäosien palautusta suoritetaan..."

#: ../src/MountManager.py:238
msgid "Please wait while scanning for devices..."
msgstr "Odota. Etsitään laitteita..."

#: ../src/MountManager.py:346
#, fuzzy
msgid "Please wait while scanning your receiver devices..."
msgstr "Odota. Etsitään laitteita..."

#: ../src/RestoreWizard.py:195
msgid "Please wait while settings restore completes..."
msgstr "Odota, asetusten palautus kesken..."

#: ../src/SoftcamManager.py:286
msgid "Please wait while starting\n"
msgstr "Odota, käynnistetään\n"

#: ../src/SoftcamManager.py:384
msgid "Please wait while stopping\n"
msgstr "Odota, pysäytetään\n"

#: ../src/ImageManager.py:416
msgid "Please wait while the flash prepares."
msgstr "Odota kun asennusta valmistellaan."

#: ../src/RestoreWizard.py:128 ../src/RestoreWizard.py:133
msgid "Please wait while the system gathers information..."
msgstr "Ole hyvä ja odota, järjestelmä kerää tietoja..."

#: ../src/MountManager.py:370
msgid "Please wait."
msgstr "Odota."

#: ../src/BackupManager.py:1237
msgid "Plugin listing failed - e. g. wrong backup destination or no space left on backup device."
msgstr "Pluginien listaus epäonnistui - väärä varmuuskopion sijainti tai levytila on lopussa."

#: ../src/BackupManager.py:1126 ../src/BackupManager.py:1130
msgid "Preparing extra plugins..."
msgstr "Valmistellaan ylimääräisiä lisäosia..."

#: ../src/BackupManager.py:216 ../src/ImageManager.py:221
msgid "Press 'Menu' to select a storage device"
msgstr ""

#: ../src/IPKInstaller.py:180
msgid "Press OK to toggle the selection."
msgstr "Suorita valinta OK-näppäimellä."

#: ../src/H9SDmanager.py:32
msgid "Press appropiate Init to move Nand root to SDcard."
msgstr "Paina sopivaa alustusta siirtääksesi Nand root SD-kortille."

#: ../src/H9SDmanager.py:33
msgid "Reboot"
msgstr "Uudelleenkäynnistys"

#: ../src/BackupManager.py:438
msgid "Rebooting..."
msgstr "Käynnistyy uudelleen..."

#: ../src/ImageManager.py:369
#, python-format
msgid ""
"Recording(s) are in progress or coming up in few seconds!\n"
"Do you still want to flash image\n"
"%s?"
msgstr ""
"Tallennus käynnissä, tai alkamassa hetken kuluttua\n"
"Haluatko silti päivittää imagen\n"
"%s?"

#: ../src/Multibootmgr.py:81
#, fuzzy, python-format
msgid "Removal of this slot will not show in %s Gui.  Are you sure you want to delete image slot?"
msgstr "Tämän paikan poisto ei näy %s käyttöliittymässä. Oletko varma että haluat poistaa imagen paikasta %s ?"

#: ../src/BackupManager.py:302 ../src/ImageManager.py:301
#: ../src/Multibootmgr.py:83 ../src/Multibootmgr.py:87
msgid "Remove confirmation"
msgstr "Poista vahvistus"

#: ../src/ImageManager.py:775 ../src/ImageManager.py:779
msgid "Removing temp mounts..."
msgstr "Poistetaan väliaikaiset liitokset..."

#: ../src/SoftcamManager.py:142
msgid "Restart"
msgstr "Käynnistä uudelleen"

#: ../src/IPKInstaller.py:139
msgid "Restart GUI."
msgstr "Käynnistä käyttöliittymä uudelleen."

#: ../src/MountManager.py:375
#, fuzzy
msgid "Restart receiver"
msgstr "Käynnistä uudelleen"

#: ../src/BackupManager.py:142
msgid "Restore"
msgstr "Palauta"

#: ../src/BackupManager.py:365
#, fuzzy
msgid "Restore confirmation"
msgstr "Palautuksen varmistus"

#: ../src/plugin.py:174 ../src/RestoreWizard.py:129 ../src/RestoreWizard.py:134
#: ../src/RestoreWizard.py:141 ../src/RestoreWizard.py:147
#: ../src/RestoreWizard.py:153 ../src/RestoreWizard.py:157
#: ../src/RestoreWizard.py:183 ../src/RestoreWizard.py:187
#: ../src/RestoreWizard.py:196 ../src/RestoreWizard.py:252
#: ../src/RestoreWizard.py:256 ../src/RestoreWizard.py:260
#: ../src/RestoreWizard.py:264 ../src/RestoreWizard.py:268
msgid "Restore wizard"
msgstr "Palautuksen avustaja"

#: ../src/BackupManager.py:394 ../src/BackupManager.py:398
#: ../src/BackupManager.py:402
msgid "Restoring backup..."
msgstr "Palautetaan varmuuskopiota..."

#: ../src/BackupManager.py:430 ../src/BackupManager.py:434
msgid "Restoring plugins, this can take a long time..."
msgstr "Palautetaan lisäosia, tämä voi kestää kauan..."

#: ../src/BackupManager.py:422 ../src/BackupManager.py:426
msgid "Restoring plugins..."
msgstr "Palautetaan lisäosia..."

#: ../src/Multibootmgr.py:41
msgid "Retrieving image slots - Please wait..."
msgstr "Haetaan image-paikkoja - Odota..."

#: ../src/ScriptRunner.py:47
msgid "Run"
msgstr "Suorita"

#: ../src/ui.py:52
msgid "Run your shell scripts."
msgstr "Suorita komentorivin käskyjonoja (scripts)."

#: ../src/MountManager.py:102
msgid "SDCARD: "
msgstr "SD-Kortti: "

#: ../src/SwapManager.py:262
msgid "SWAP file not found. You have to create the file before you try to activate it."
msgstr "SWAP-tiedostoa ei löytynyt. Luo se ensin, ennen kuin yrität aktivoida sitä."

#: ../src/ui.py:53 ../src/SwapManager.py:107
msgid "SWAP manager"
msgstr "SWAP-tiedoston hallinnointi"

#: ../src/SwapManager.py:112
msgid "SWAP place:"
msgstr "Paikka SWAP:lle:"

#: ../src/SwapManager.py:114
msgid "SWAP size:"
msgstr "SWAP-koko:"

#: ../src/BackupManager.py:720 ../src/BackupManager.py:809
#: ../src/MountManager.py:336
msgid "Save"
msgstr "Tallenna"

#: ../src/ui.py:52 ../src/plugin.py:168 ../src/ScriptRunner.py:45
msgid "Script runner"
msgstr "Skriptien ajo"

#: ../src/BackupManager.py:755
msgid "Select"
msgstr "Valitse"

#: ../src/BackupManager.py:232 ../src/BackupManager.py:235
msgid "Select a backup to restore:"
msgstr "Valitse palautettava varmuskopio:"

#: ../src/IPKInstaller.py:106
msgid "Select a package to install:"
msgstr "Valitse asennettava ohjelmapaketti:"

#: ../src/ImageManager.py:1319
#, fuzzy
msgid "Select an image to download:"
msgstr "Valitse päivitettävä image:"

#: ../src/ImageManager.py:239 ../src/ImageManager.py:244
msgid "Select an image to flash:"
msgstr "Valitse päivitettävä image:"

#: ../src/BackupManager.py:806
msgid "Select extra packages folder"
msgstr "Valitse lisäohjelmapakettien hakemisto"

#: ../src/BackupManager.py:717
msgid "Select files/folders to backup"
msgstr "Valitse varmuuskopioitavat tiedostot/kansiot"

#: ../src/BackupManager.py:838
msgid "Select folder that contains plugins"
msgstr "Valitse hakemisto joka sisältää lisäosat"

#: ../src/SwapManager.py:303
msgid "Select the SWAP file size:"
msgstr "Valitse SWAP:n koko:"

#: ../src/SoftcamManager.py:82
msgid "Select:"
msgstr "Valitse:"

#: ../src/ImageManager.py:739
msgid "Setting up..."
msgstr "Asennus käynnissä..."

#: ../src/MountManager.py:198
msgid "Setup mounts"
msgstr "Aseta liitokset"

#: ../src/MountManager.py:123 ../src/MountManager.py:129
#: ../src/MountManager.py:131 ../src/MountManager.py:133
#: ../src/MountManager.py:135
msgid "Size: "
msgstr "Koko: "

#: ../src/plugin.py:99 ../src/plugin.py:166 ../src/SoftcamManager.py:80
msgid "Softcam manager"
msgstr "Softcam:n hallinnointi"

#: ../src/SoftcamManager.py:284
msgid "Softcam starting..."
msgstr "Softcam käynnistyy..."

#: ../src/SoftcamManager.py:382
msgid "Softcam stopping..."
msgstr "Softcam pysähtymässä..."

#: ../src/SoftcamManager.py:555
msgid "SoftcamCheck"
msgstr "Softcam-tarkistus"

#: ../src/IPKInstaller.py:90
msgid "Sorry but that location does not exist or is not setup. Please set it up in the Backup manager setup menu."
msgstr "Tämä sijainti ei ole olemassa, tai sitä ei ole asetettu. Aseta se Varmuuskopioiden hallinnointityökalulla."

#: ../src/BackupManager.py:550
msgid "Sorry the feeds are down for maintenance. Please try again later."
msgstr "Syötteitä huolletaan, kokeile myöhemmin uudestaan."

#: ../src/RestoreWizard.py:146
msgid "Sorry the feeds are down for maintenance. Please try using Backup manager to restore plugins later."
msgstr "Syötteitä huolletaan. Yritä myöhemmin palauttaa pluginit käyttäen varmuuskopioiden hallinta-työkalua."

#: ../src/RestoreWizard.py:263
msgid "Sorry the feeds are down for maintenance. Please try using the Backup manager to restore plugins later."
msgstr "Syötteitä huolletaan. Yritä myöhemmin palauttaa pluginit käyttäen varmuuskopioiden hallinta-työkalua."

#: ../src/BackupManager.py:358 ../src/BackupManager.py:360
#: ../src/RestoreWizard.py:182 ../src/RestoreWizard.py:186
msgid "Sorry, but the file is not compatible with this image version."
msgstr "Tiedosto ei ole yhteensopiva tämän ohjelmaversion kanssa."

#: ../src/BackupManager.py:477
msgid "Sorry, but the restore failed."
msgstr "Palautus epäonnistui."

#: ../src/SwapManager.py:297
msgid "Sorry, no physical devices that supports SWAP attached. Can't create SWAP file on network or fat32 file-systems."
msgstr "Fyysisiä laitteita, jotka tukevat SWAP-tiedostoa, ei ole. SWAP-tiedostoa ei pysty luomaan verkkolevylle tai FAT32-levyjärjestelmään."

#: ../src/ImageManager.py:859
msgid "Sorry, not enough free RAM found, and no physical devices that supports SWAP attached. Can't create SWAP file on network or fat32 file-systems, unable to make backup."
msgstr "Vapaata RAM-muistia ei ole tarpeeksi, eikä fyysistä laitetta löydy, joka tukisi SWAP-tiedostoa. SWAP-tiedostoa ei voida luoda verkkolevylle tai FAT32-tiedostojärjestelmään. Varmuuskopiota ei pysty tekemään."

#: ../src/SoftcamManager.py:136
msgid "Start"
msgstr "Käynnistä"

#: ../src/BackupManager.py:1102 ../src/BackupManager.py:1106
msgid "Starting..."
msgstr "Käynnistyy..."

#: ../src/SwapManager.py:116
msgid "Status:"
msgstr "Tila:"

#: ../src/SoftcamManager.py:138
msgid "Stop"
msgstr "Pysäytä"

#: ../src/MountManager.py:129
msgid "TB"
msgstr "TB"

#: ../src/IPKInstaller.py:96 ../src/IPKInstaller.py:104
msgid "Temp folder"
msgstr "Temp -hakemisto"

#: ../src/BackupManager.py:1197
msgid "The backup location does not have enough free space."
msgstr "Varmuuskopioiden tallennuskohteessa ei ole riittävästi tilaa."

#: ../src/ImageManager.py:821
msgid "The backup location does not have enough free space.\n"
msgstr "Varmuuskopioiden tallennuskohteessa ei ole riittävästi tilaa.\n"

#: ../src/MountManager.py:373
#, fuzzy
msgid ""
"The changes need a system restart to take effect.\n"
"Restart your receiver now?"
msgstr ""
"E2 on uudelleenkäynnistettävä.\n"
"Uudelleenkäynnistä%s %s nyt?"

#: ../src/BackupManager.py:232 ../src/ImageManager.py:239
msgid "The chosen location does not exist, using /media/hdd."
msgstr "Valittua hakemistoa ei ole, käytetään /media/hdd."

#: ../src/BackupManager.py:255 ../src/ImageManager.py:253
msgid "There is a problem with this device. Please reformat it and try again."
msgstr "Tässä laitteessa on ongelma. Alusta se ja yritä uudelleen."

#: ../src/BackupManager.py:304
msgid "There is no backup to delete."
msgstr "Ei ole varmuuskopiota mitä poistaa."

#: ../src/BackupManager.py:345
msgid "There is no backup to restore."
msgstr "Ei ole varmuuskopiota mitä palauttaa."

#: ../src/ImageManager.py:303
msgid "There is no image to delete."
msgstr "Ei ole imagea mitä poistaa."

#: ../src/ImageManager.py:411
msgid "There is no image to flash."
msgstr "Ei ole imagea mitä päivittää."

#: ../src/MountManager.py:299
msgid "This device is already mounted as HDD."
msgstr "Tämä laite on jo liitetty kovalevyksi."

#: ../src/MountManager.py:144
msgid "Type: "
msgstr "Tyyppi: "

#: ../src/MountManager.py:96
msgid "USB: "
msgstr "USB: "

#: ../src/MountManager.py:199
msgid "Un-mount"
msgstr "Poista liitos"

#: ../src/ImageManager.py:442
#, python-format
msgid ""
"Unzip error (also sent to any debug log):\n"
"%s"
msgstr ""
"Unzip-virhe (lähetetty debug-lokiin):\n"
"%s"

#: ../src/MountManager.py:368
msgid "Updating mount locations..."
msgstr "Päivitetään liitos-paikkoja..."

#: ../src/MountManager.py:222
msgid "Use as HDD"
msgstr "Käytä HDD:nä"

#: ../src/Multibootmgr.py:37
msgid "Use the cursor keys to select an installed image and then Erase button."
msgstr "Käytä kursoria valitaksesi asennetun imagen, ja paina Pyyhi-nappulaa."

#: ../src/BackupManager.py:187 ../src/BackupManager.py:328
#: ../src/BackupManager.py:371 ../src/ImageManager.py:174
#: ../src/ImageManager.py:337
msgid "View progress"
msgstr "Tarkastele etenemistä"

#: ../src/plugin.py:84 ../src/plugin.py:163
msgid "Vision"
msgstr "Vision"

#: ../src/plugin.py:176
msgid "Vision Backup manager"
msgstr "Vision Backup manager"

#: ../src/ui.py:44
#, fuzzy
msgid "Vision Core"
msgstr "Vision"

#: ../src/plugin.py:177 ../src/ImageManager.py:125
msgid "Vision Image manager"
msgstr "Vision Image manager"

#: ../src/plugin.py:178
msgid "Vision Mount manager"
msgstr "Vision Mount manager"

#: ../src/plugin.py:180
msgid "Vision SWAP manager"
msgstr "Vision SWAP-hallinnointi"

#: ../src/plugin.py:179
msgid "Vision Script runner"
msgstr "Vision Script runner"

#: ../src/SwapManager.py:308
msgid "Wait please while creating SWAP file..."
msgstr "Odota kun SWAP-tiedostoa luodaan..."

#: ../src/SoftcamManager.py:168
msgid ""
"Wait please while scanning\n"
"for softcam's..."
msgstr ""
"Odota, haetaan\n"
"softcammeja..."

#: ../src/SwapManager.py:146
msgid "Wait please while scanning..."
msgstr "Odota, tietoja haetaan..."

#: ../src/BackupManager.py:52 ../src/ImageManager.py:62
msgid "Weekly"
msgstr "Viikoittain"

#: ../src/RestoreWizard.py:100 ../src/RestoreWizard.py:103
msgid "YES, to restore plugins"
msgstr "KYLLÄ, palauttaa lisäosat"

#: ../src/RestoreWizard.py:94
msgid "YES, to restore settings"
msgstr "KYLLÄ, palauttaa asetukset"

#: ../src/ImageManager.py:373
msgid ""
"You are about to flash an eMMC flash; we cannot take any responsibility for any errors or damage to your box during this process.\n"
"Proceed with CAUTION!:\n"
"Are you sure you want to flash this image:\n"
" "
msgstr ""
"Olet tekemässä eMMC-päivitystä. Emme ota vastuuta, jos laitteesi rikkoutuu. Etene huolellisesti!! Oletko varma että haluat päivittää.\n"
"Proceed with CAUTION!:\n"
"Are you sure you want to flash this image:\n"
" "

#: ../src/ImageManager.py:413
msgid "You have decided not to flash image."
msgstr "Olet päättänyt olla päivittämättä imagea."

#: ../src/SwapManager.py:328
msgid "You have to create a SWAP file before trying to activate the autostart."
msgstr "Sinun pitää ensin luoda SWAP-tiedosto, ennen kuin aktivoit sen automaattisen käynnistyksen."

#: ../src/RestoreWizard.py:255
#, fuzzy
msgid "Your receiver could not connect to the plugin feeds at this time. Please try using the Backup manager to restore plugins later."
msgstr "Sinun %s %s ei saa yhteyttä plugin-syötteisiin. Yritä myöhemmin palauttaa pluginit käyttäen varmuuskopioiden hallinta-työkalua."

#: ../src/ImageManager.py:596
#, fuzzy
msgid ""
"Your receiver is about to create a full image backup, this can take about 6 minutes to complete.\n"
"Do you want to allow this?"
msgstr ""
"Sinun %s %s aikoo tehdä täyden varmuuskopion, ja se voi kestää noin kuusi minuuttia.\n"
"Sallitko tämän?"

#: ../src/BackupManager.py:1039
#, fuzzy
msgid ""
"Your receiver is about to run a backup of your settings and to detect your plugins.\n"
"Do you want to allow this?"
msgstr ""
"Sinun %s %s ajaa varmuuskopion asetuksistasi, ja yrittää havaita pluginisi.\n"
"Sallitko tämän?"

#: ../src/BackupManager.py:541
#, fuzzy
msgid "Your receiver is not connected to a network. Please check your network settings and try again."
msgstr "Sinun %s %s ei saa yhteyttä verkkoon. Tarkista verkkoasetuksesi, ja kokeile uudestaan."

#: ../src/RestoreWizard.py:251
#, fuzzy
msgid "Your receiver is not connected to a network. Please try using the Backup manager to restore plugins later when a network connection is available."
msgstr "Sinun %s %s ei saa yhteyttä verkkoon. Yritä myöhemmin palauttaa pluginit käyttäen varmuuskopioiden hallinta-työkalua, kun verkkoyhteys on saatavilla."

#: ../src/BackupManager.py:559
#, fuzzy
msgid "Your receiver is not connected to the Internet. Please check your network settings and try again."
msgstr "Sinun %s %s ei ole yhteydessä internettiin. Tarkasta verkkoasetukset ja kokeile uudestaan."

#: ../src/RestoreWizard.py:152
#, fuzzy
msgid "Your receiver is not connected to the Internet. Please try using Backup manager to restore plugins later."
msgstr "Sinun %s %s ei saa yhteyttä internettiin. Yritä myöhemmin palauttaa pluginit käyttäen varmuuskopioiden hallinta-työkalua."

#: ../src/RestoreWizard.py:259
#, fuzzy
msgid "Your receiver is not connected to the Internet. Please try using the Backup manager to restore plugins later."
msgstr "Sinun %s %s ei saa yhteyttä internettiin. Yritä myöhemmin palauttaa pluginit käyttäen varmuuskopioiden hallinta-työkalua."

#: ../src/ImageManager.py:494
#, fuzzy, python-format
msgid ""
"ofgwrite error (also sent to any debug log):\n"
"%s"
msgstr ""
"OFGwrite virhe (virhe lähetetty debug-lokiin):\n"
"%s"

#: ../src/ImageManager.py:395
#, python-format
msgid "slot%s - %s"
msgstr "paikka%s - %s"

#: ../src/Multibootmgr.py:73
#, python-format
msgid "slot%s - %s "
msgstr "paikka%s - %s "

#: ../src/ImageManager.py:395
#, python-format
msgid "slot%s - %s (current image)"
msgstr "paikka%s - %s (nykyinen image)"

#: ../src/MountManager.py:110 ../src/MountManager.py:123
#: ../src/MountManager.py:135
msgid "unavailable"
msgstr "ei saatavilla"
